packages/vitest/src/constants.ts

```diff
@@ -49,6 +49,7 @@ export const configDefaults: UserConfig = Object.freeze({
   isolate: true,
   watchIgnore: [/\/node_modules\//, /\/dist\//],
   update: false,
+  run: !!process.env.CI,
```

Comment:
The fix should be make watch default to false and handles it in CLI, /cc @sheremet-va

Reply:
Should be fixed